gen/bundle/bundle.go

```diff
@@ -89,14 +96,3 @@ func main() {
 		panic(err)
 	}
 }
-
-func getOldGitTagFromEmbeddedMetadata(m *build.BuiltinActorsMetadata) string {
-	for _, v := range build.EmbeddedBuiltinActorsMetadata {
-		// if we agree on the manifestCid for the previously embedded metadata, use the previously set tag
-		if m.Version == v.Version && m.Network == v.Network && m.ManifestCid == v.ManifestCid {
-			return m.BundleGitTag
```

Comment:
This was a bug, should have been `v.BundleGitTag` which is why the `BundleGitTag`s have been progressively disappearing from builtin_actors_gen.go each time we upgrade. I decided to just roll it in to the above code while I was fixing it.